eth/tracers/internal/tracers/prestate_tracer.js

```diff
@@ -85,7 +85,7 @@
 			case "EXTCODECOPY": case "EXTCODESIZE": case "BALANCE":
 				this.lookupAccount(toAddress(log.stack.peek(0).toString(16)), db);
 				break;
-			case "CREATE":
+			case "CREATE": case "CREATE2":
```

Comment:
This is not correct. The code below uses `toContract(from, db.getNonce(from))` to generate the contract address, but CREATE2 has a different contract address generation scheme. You'll need to implement a `toContract2` similar to https://github.com/ethereum/go-ethereum/blob/master/eth/tracers/tracer.go#L354